### PR TITLE
E621 Command

### LINKED ISSUE #78: Add an e621 command

### PR DESCRIPTION
**Pull request by Doges**
Add the E621 Command (NSFW) 

# Closing threads

Closes #78

### CODE REVIEW

**Comment by Doges** [comment]
ready to review

File: src/locales/en-US/commands.json [hunk 1]
```diff
@@ -1,179 +1,182 @@
...
+  },
+  "setlanguage": {
+    "commandDescription": "Changes the bot's language",
+    "commandUsage": "<language code>",
+    "noCode": "You need to give me a language code.",
+    "availableLanguages": "Available languages",
+    "changedSuccessfully": "Language changed successfully to {{lang}}!"
+  },
+  "e621": {
+    "hereIsYour_yiff": "Here's your yiff!"
```
Comment (pedrofracassi) [inline src/locales/en-US/commands.json]: No need to use context (`_`) here.

**Comment by bolsomito** [comment]
can you just `git pull origin master` and commit that?

File: package.json [hunk 2]
```diff
@@ -27,14 +27,15 @@
   "homepage": "https://github.com/SwitchbladeBot/switchblade#readme",
   "dependencies": {
     "api-npm": "^1.0.0",
+    "booru": "^0.4.0",
     "bufferutil": "^3.0.5",
     "crowdin": "^1.0.0",
     "discord.js": "^11.3.2",
     "discord.js-lavalink": "^2.0.2",
     "erlpack": "github:discordapp/erlpack",
+    "googleapis": "^31.0.2",
```
Comment (pedrofracassi) [inline package.json]: Is this even used?

**Comment by bolsomito** [comment]
nvm, nvm

File: package.json (hunk 2 above)
Comment (bolsomito) [reply package.json]: it is, for s!np i believe

Comment (pedrofracassi) [reply package.json]: oh ok

File: src/locales/en-US/commands.json (hunk 1 above)
Comment (pedrofracassi) [reply src/locales/en-US/commands.json]: https://www.i18next.com/translation-function/context

Comment (Doges) [reply src/locales/en-US/commands.json]: than you don't need the context in neko command, stfu and approve this, if you don't approve i will open an issue for you.

**Pull request merged**